subsys/bluetooth/host/Kconfig

```diff
@@ -734,6 +734,14 @@ config BT_CONN_PARAM_RETRY_TIMEOUT
 	  The value is a timeout used by peripheral device to wait until retry
 	  to attempt requesting again the preferred connection parameters.
 
+config BT_TRANSMIT_POWER_CONTROL
```

Comment:
why do we need it?

Reply:
I assumed this is used to enable the feature for usage in host. When testing with the bt shell it works without this also though. I removed it.

Reply:
When adding controller feature we generally have three kconfigs.
- One for the controller to mark support
- One for enabling support in the controller
- One for enabling support in the host

If we are not planning to upstream this we might not need all the kconfigs, but it would be nice to keep the same setup as the other features.

Reply:
I've updated it to use the host kconfig. The issue was that I used the ctrl one to compile in host code. This is now amended. I tested this again in bt shell. Will ask Carles if there is anything else I need to add/amend for kconfigs. kconfigs (ctrl ones) are in sdk-nrf and ofc sdc.

Reply:
I've updated the controller kconfigs. Now only the host config needs to be selected for the feature to work, i.e. to automatically also select the controller config. Please take a look if the kconfigs seem good now, and if there is anything else needing amendment in this PR.